tests/test_parsing/test_parse_pddl.py

```diff
@@ -0,0 +1,18 @@
+from lisdf.parsing.parse_pddl import parse_pddl
+
+
+def test_parse_pddl():
+    init_atoms, goal_atoms, type_to_obj_dict = parse_pddl(
+        "examples/pddl/clutter.pddl", "examples/pddl/clutter_prob01.pddl"
+    )
+    for atom in init_atoms:
+        # Test that the 0th element of the atoms, which should represent
+        # the predicate names, is in the list of known predicate names
+        assert atom[0] in ["on-surface", "arm-empty", "holding", "on"]
```

Comment:
Can you add check that none of the other elements in the list are in the predicate names. Something like

```
assert not any(elem in predicate_names for elem in atom[1:])
```